jobs/JGFS_ATMOS_VMINMON

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env bash
+
+###########################################################
+# GFS Minimization Monitor (MinMon) job
+###########################################################
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "vrfy" -c "base vrfy"
```

Comment:
Same comment as above, need to use the v16 versions with any needed changes for GSI `develop`.